domain-ee/ee-ep-merge-app/integration/conftest.py

```diff
@@ -0,0 +1,50 @@
+import pytest_asyncio
+from integration.mq_endpoint import MqEndpoint
+from src.python_src.config import QUEUES, REPLY_QUEUES, ClientName
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+def get_claim_contentions_endpoint():
+    name = ClientName.GET_CLAIM_CONTENTIONS
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def put_tsoj_endpoint():
+    name = ClientName.PUT_TSOJ
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def update_claim_contentions_endpoint():
+    name = ClientName.UPDATE_CLAIM_CONTENTIONS
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def cancel_claim_endpoint():
+    name = ClientName.CANCEL_CLAIM
+    endpoint = MqEndpoint(name, QUEUES[name], REPLY_QUEUES[name])
+    return endpoint
+
+
+@pytest_asyncio.fixture(autouse=True, scope="session")
+async def endpoint_lifecycle(event_loop,
```

Comment:
I think I'm not understanding something about pytest_asyncio fixtures, but, where does `endpoint_lifecycle` actually get used?

Reply:
Ah. So this is a yield fixture. Think set up and tear down at the session scope. See [here](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended).

Reply:
Got it, so the `autouse=True, scope="session"` tells it to automatically get setup/torn-down when the whole pytest session starts/ends.